cmake/jinja.cmake

```diff
@@ -1,15 +1,18 @@
-#-
-# =======================================================================
-# Copyright 2018 Autodesk, Inc. All rights reserved.
 #
-# This computer source code and related instructions and comments are the
-# unpublished confidential  and proprietary information of Autodesk, Inc.
-# and are protected under applicable copyright and trade secret law. They 
-# may not be disclosed to, copied  or used by any third party without the 
-# prior written consent of Autodesk, Inc.
-# =======================================================================
-#+
-
+# Copyright 2020 Autodesk
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
```

Comment:
Keeping the Copyright style consistence.

Also,we don't need to add Copyright headers in every Cmake files ( it is very distracting ). Only add in the top level cmake file.